README.md

```diff
@@ -1 +1,18 @@
-# portfolio
\ No newline at end of file
+Portfolio: setup and mobile version skeleton 
+My second microverse project.
+
+Built With HTML & CSS Visual studio code
+
+Authors 👤 
+Imran Belete 
+GitHub: @ibel251 
+Twitter: ibbimran7@gmail.com 
+LinkedIn: ibbimran7@gmail.com
+
+🤝 Contributing Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a ⭐️ if you like this project!
```

Comment:
- [ ]  Your README looks almost empty of content, but I think you can improve it by making it more professional. A professional README file should have a project title, project description, list of technology used in the project, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.